src/v/redpanda/tests/fixture.h

```diff
@@ -57,28 +76,68 @@ class redpanda_thread_fixture {
           app.id_allocator_frontend);
     }
 
+    // creates single node with default configuration
+    redpanda_thread_fixture()
+      : redpanda_thread_fixture(
+        model::node_id(1),
+        9092,
+        33145,
+        43118,
+        43189,
+        {},
+        fmt::format("test.dir_{}", time(0)),
+        scheduling_groups{},
```

Comment:
do we not need to create the groups some how like is done in application.cc in the default case

```
    _scheduling_groups.create_groups().get();
```

or was maybe the intention here to pass in nullopt?

Reply:
great catch, fixed